benchmark_db_terraform/modules/benchmark_db/service/Gemfile.lock

```diff
@@ -0,0 +1,36 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    citrus (3.0.2)
+    concurrent-ruby (1.2.2)
+    csv (3.2.6)
+    deep_merge (1.2.2)
+    fileutils (1.7.0)
+    logger (1.5.3)
+    open3 (0.1.2)
+    optparse (0.3.1)
+    set (1.0.3)
+    tmpdir (0.1.3)
+      fileutils
+    toml-rb (2.2.0)
+      citrus (~> 3.0, > 3.0)
+    tomlrb (2.0.3)
+
+PLATFORMS
+  arm64-darwin-21
```

Comment:
This is not the platform of the DO droplets.

Reply:
fixed